### ISSUE TITLE
Where is conf_yaml?

### ISSUE THREAD

**Issue by JiahangOK** [JustinLin610/LancoSum#1]
Excuse me, I want to try your codes, but when I begin to train, I find that there is no conf_yaml, so I have no config file. Could you please let me see your config file?

**Comment by lamho-bienetre** [comment]
data: '.../'
logF: '.../'
epoch: 20
batch_size: 64
optim: 'adam'
cell: 'lstm'
attention: 'luong_gate'
learning_rate: 0.0003
max_grad_norm: 10
learning_rate_decay: 0.5
start_decay_at: 6
emb_size: 512
hidden_size: 512
dec_num_layers: 3
enc_num_layers: 3
bidirectional: True
dropout: 0.0
max_time_step: 50
eval_interval: 10000
save_interval: 3000
metrics: ['rouge']
shared_vocab: True
beam_size: 10
unk: True
schedule: False
selfatt: True
schesamp: False
swish: True
length_norm: True
gated: True

---
I create a conf_yaml file like above, it works!
